src/components/invoice/index.js

```diff
@@ -1,21 +1,140 @@
-        <tr>
-          <th className="invoiceCabinName">Cabin Name</th>
-          <th className="invoiceSize">Size</th>
-          <th className="invoicePrice">Price</th>
-        </tr>
-        <tr>
-          <td>
-            {selectedCabinName === '' ? 'select cabin' : selectedCabinName}
-          </td>
-          <td>Size</td>
-          <td>£1000</td>
-        </tr>
+        <thead>
+          <tr>
+            <th className="invoiceCabinName">Cabin Name</th>
+            <th className="invoiceWallThickness">Wall Thickness</th>
+            <th className="invoiceSize">Size</th>
+            <th className="invoicePrice">Price</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td>
+              {selectedCabinName === '' ? 'select cabin' : selectedCabinName}
+            </td>
+            <td>{selectedWallOpt}mm</td>
+            <td>
+              {selectedSize === '' ? 'Select Size ' : selectedSize.width}m x
+              {selectedSize.depth}m
+            </td>
+            <td>£{cabinPrice[0] !== undefined && cabinPrice[0].toFixed(2)}</td>
+          </tr>
+          <tr>
+            <th colSpan={4}>Optional Extras</th>
+          </tr>
+          {selectedVeranda && (
+            <tr>
+              <td colSpan={3}>{'Veranda and Base'}</td>
+              <td>£{verandaPrice[0].toFixed(2)}</td>
+            </tr>
+          )}
+          {selectedRoofInsulation && (
+            <tr>
+              <td colSpan={3}>{'Roof Insulation'}</td>
+              <td>£{roofInsulationPrice[0].toFixed(2)}</td>
+            </tr>
+          )}
+          {selectedFloorInsulation && (
+            <tr>
+              <td colSpan={3}>{'Floor Insulation'}</td>
+              <td>£{floorInsulationPrice[0].toFixed(2)}</td>
+            </tr>
+          )}
+          {selectedWallInsulation && (
+            <tr>
+              <td colSpan={3}>{'Wall Insulation'}</td>
+              <td>£{wallInsulationPrice[0].toFixed(2)}</td>
+            </tr>
+          )}
+          {selectedBaseWork && (
+            <tr>
+              <td colSpan={3}>{'Cabin Structural Base'}</td>
+              <td>£{baseWorkPrice[0].toFixed(2)}</td>
+            </tr>
+          )}
+          {selectedCabinInstallation && (
+            <tr>
+              <td colSpan={3}>{'Cabin Installation'}</td>
+              <td>£{cabinInstallationPrice[0].toFixed(2)}</td>
+            </tr>
+          )}
+          {selectedAirCon && (
+            <tr>
+              <td colSpan={3}>{'Heating & Air-Con'}</td>
+              <td>£{airConPrice.toFixed(2)}</td>
+            </tr>
+          )}
+          <tr>
+            <td colSpan={3}>Total</td>
+            <td>£{total.toFixed(2)}</td>
+          </tr>
+          <tr>
+            <td colSpan={3}>Deposit</td>
+            <td>£{deposit.toFixed(2)}</td>
+          </tr>
+          <tr>
+            <td colSpan={3}>Balance</td>
+            <td>£{balance.toFixed(2)}</td>
+          </tr>
```

Comment:
oh i spotted one thing - you should get in to the habit of using `&pound;` instead of £ (and any other symbol - use [HTML entities instead](https://www.w3schools.com/html/html_entities.asp)) because although your browser might show them correctly, not everyone's will.  Using html entities ensures they displayed correctly